cmd/parca-agent/main.go

```diff
@@ -108,8 +108,11 @@ type flags struct {
 	DebuginfoStrip               bool          `kong:"help='Only upload information needed for symbolization. If false the exact binary the agent sees will be uploaded unmodified.',default='true'"`
 	DebuginfoUploadCacheDuration time.Duration `kong:"help='The duration to cache debuginfo upload exists checks for.',default='5m'"`
 
+	// Hidden debug flags (only for debugging):
+	DebugProcessNames []string `kong:"help='Only attach profilers to specified processes. comm name will be used to match the given matchers. Accepts Go regex syntax (https://pkg.go.dev/regexp/syntax).',hidden=''"`
```

Comment:
Do we really need this to be a list? A single regex would be sufficient, no? If people really want to they can just use "or" in the regex.

Reply:
This is true. I also wanted to give the option to match against the exact names without thinking of the regex syntax. e.g. `etcd, kube-apiserver`